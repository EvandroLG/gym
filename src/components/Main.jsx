import React, { Component } from 'react';
import { BrowserRouter, Route } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../store';
import Menu from './Menu';
import TrainingList from './training_list/Index';
import NewTraining from './new_training/Index';

import './Main.css';

export default class Main extends Component {
  render() {
    return (
      <BrowserRouter>
        <Provider store={store}>
          <div>
            <Menu />
            <div className="content">
              <Route exact path="/" component={TrainingList} />
              <Route path="/new_training" component={NewTraining} />
            </div>
          </div>
        </Provider>
      </BrowserRouter>
    )
  }
}
