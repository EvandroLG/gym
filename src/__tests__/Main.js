import React from 'react';
import { shallow } from 'enzyme';
import Main from '../components/Main';

/* global verifySnapshot */

describe('main', () => {

  it('should render as expected', () => {
    const component = shallow(<Main />);
    verifySnapshot(component);
  });
});
