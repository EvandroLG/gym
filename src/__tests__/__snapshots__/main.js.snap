// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main should render as expected 1`] = `
<BrowserRouter>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <div>
      <Menu />
      <div
        className="content"
      >
        <Route
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          path="/new_training"
        />
      </div>
    </div>
  </Provider>
</BrowserRouter>
`;
